blackboxtest/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "condenser-blackboxtest",
+  "version": "1.0.0",
+  "main": "false",
```

Comment:
Why is main set to false and why is it a string?

Reply:
good point